### DISCUSSION TITLE
ente auth tile (Android)

### DISCUSSION THREAD

**Discussion by ghost** [ente-io/ente#2266] [Enhancements]
Hi,
it would be cool if tiles can be added for ente auth on android.

This would allow the ente auth app to be opened much faster by swiping down and tap on the tile.

**Comment by haritvsmurali** [comment]
Quick Settings Tile for Android & Control Center Shortcut in iOS.

**Comment by Miltora** [comment]
any updates, as I have seen this in 2 other discussions?
